tests/terraform/scripts/delete_resources.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+#Get resource name from tfvarslocal && change name to make more sense in this context
+RESOURCE_NAME=$(grep resource_name <tests/terraform/modules/config/local.tfvars | cut -d= -f2 | tr -d ' "')
+NAME_PREFIX="$RESOURCE_NAME"
+
+##Terminate the instances
+echo "Terminating instances for $NAME_PREFIX if it is still running"
+# shellcheck disable=SC2046
+aws ec2 terminate-instances --instance-ids $(aws ec2 describe-instances \
+  --filters "Name=tag:Name,Values=${NAME_PREFIX}*" \
+  "Name=instance-state-name,Values=running" --query \
+  'Reservations[].Instances[].InstanceId' --output text) > /dev/null 2>&1
+
+#Get the list of load balancer ARNs
+LB_ARN_LIST=$(aws elbv2 describe-load-balancers \
+  --query "LoadBalancers[?starts_with(LoadBalancerName, '${NAME_PREFIX}') && Type=='network'].LoadBalancerArn" \
+  --output text)
+
+#Loop through the load balancer ARNs and delete the load balancers
+for LB_ARN in $LB_ARN_LIST; do
```

Comment:
Do we need to add a check for LB_ARN existing in LB_ARN_LIST or will the for loop is sufficient?